configdialog.cpp

```diff
@@ -45,8 +50,15 @@ BOOL ConfigDialog::command(WPARAM wparam, LPARAM lparam) {
 		}
 		case IDC_YONK_PATTERN: {
 			if (HIWORD(wparam) == CBN_SELENDOK) {
-				return combobox_changed((HWND) lparam);
+				return combobox_changed((HWND) lparam, IDC_YONK_PATTERN);
```

Comment:
You might be surprised to hear but I have no problem with how this is being done.